app/views/users/index.html.erb

```diff
@@ -4,15 +4,13 @@
   <thead>
   <tr>
     <th>Email</th>
-    <th width="100"></th>
   </tr>
   </thead>
-
   <tbody>
-  <% @users.each do |user| %>
+  <% User.all.each do |user| %>
```

Comment:
Ahhhhh, database querying in the view!

Reply:
Just copied what was on the home page before :P